Tests/UnitTests/Scripts/tests.js

```diff
@@ -1,4 +1,7 @@
-﻿// TODO: use premade reporter (once Console Polyfill is fixed)
+﻿// Set this to true to make attaching a debugging easier.
```

Comment:
Comment probably needs to be updated

Reply:
This comment is here to tell people to set this flag to true when debugging this file. Maybe I can reword it if it's not clear?

Reply:
Got it, that makes sense.